index.html

```diff
@@ -180,6 +180,21 @@ <h5><a class="page-anchor" id="gofest2023" href="#gofest2023">Meetup Event:</a>
                                  </div>
                               </p>
                            </div>
+                           <div class="asset">
+                              <a href="https://drive.google.com/file/d/1PJepFpUMkPNatfoKR5IuMBrD8uDyU88u/view?usp=drive_link">
+                              <img alt="GO Fest 2023 CDL Badge" src="./assets/2023/2023-gofest/CDLGoFest.png">
+                              <div class="asset-label">
+                                 <img src="./img/gdrive-folder-icon.png" alt="Google Drive Folder Label">
+                              </div>
+                              <label>GO Fest 2023 CDL Badge</label>
+                              </a>
+                              <p>
+                                 Go Fest 2023 CDL Badge! 
+                                 <div class="credits">
+                                    Banner Design: xLuximityx
```

Comment:
This should be updated to give the community day league seeing the asset credit, see froakie community day for example to copy paste. Only the actual credit section needs to be copied here.

Reply:
Credit updated